src/pip/_internal/req/constructors.py

```diff
@@ -115,23 +115,17 @@ def parse_editable(editable_req):
             url = f'{version_control}+{url}'
             break
 
-    if '+' not in url:
+    link = Link(url)
+
+    if not link.is_vcs:
+        backends = ", ".join([backend.name + '+' for backend in vcs.backends])
```

Comment:
```suggestion
        backends = ", ".join(f"{backend.name}+" for backend in vcs.backends)
```

Reply:
Absolutely. Thanks for the review, @xavfernandez